share/dev/windows/ocio_deps.bat

```diff
@@ -103,20 +109,24 @@ if ErrorLevel 1 (
     exit /b
 )
 
-echo Checking for Microsoft Visual Studio...
-set MSVS=0
-for /d %%a in ("%programfiles%\Microsoft Visual Studio*") do (
-    for /f "tokens=3 delims=\" %%x in ("%%a") do set MSVS=1
+IF NOT EXIST "!MSVS_PATH!" ( 
+    echo Could not find MS Visual Studio. Please provide the location for Microsoft Visual Studio vcvars64.bat or modify MSVS_PATH in the script.
+    rem The double dash are in quote here because otherwise the echo command thow an error.
+    echo "--msvs <path>"
+    echo E.g. C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build
+    exit /b
 )
 
-if !MSVS!==0 (
-    echo No Microsoft Visual Studio installation was found in !programfiles!.
-    echo For non-standard installation path, please use the following option:
-    rem The double dash are in quote here because otherwise the echo command thow an error. 
-    echo "ocio_deps --vs <path>"
+rem ****************************************************************************************************************
+rem Setting up the environment using MS Visual Studio batch script
+set VCVARS64_PATH="!MSVS_PATH!\vcvars64.bat"
```

Comment:
Looking at ocio.bat, I see that the vcvars64 script is called [later](https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/main/share/dev/windows/ocio.bat#L153) to initialize the environment. Is this needed here as well?

Reply:
Does `ocio.bat` call `ocio_deps.bat`? if not, then this is needed.